pom.xml

```diff
@@ -56,13 +56,13 @@
             </dependency>
             <dependency>
                 <groupId>edu.ucar</groupId>
-                <artifactId>netcdf</artifactId>
-                <version>4.3.22</version>
+                <artifactId>netcdf4</artifactId>
+                <version>4.6.6</version>
             </dependency>
             <dependency>
                 <groupId>edu.ucar</groupId>
                 <artifactId>opendap</artifactId>
-                <version>4.3.22</version>
+                <version>4.6.6</version>
```

Comment:
Its not an issue here as we are not attempting to write or modify netcdf 4 files using java.  The netcdf java library only requires the netcdf c library to write to netcdf4 files and then there are no versions of the netcdf java library that could work with version 4.1.1 of the c library anyway (we know we tried at one point to update attributes and we had to revert to using nccatted instead).

If we wish to write to netcdf4 files using java we would need version 4.3.1 installed or greater.